docs/api-status.json

```diff
@@ -1953,6 +1953,12 @@
         "comment": "GroupSnapGetInfo returns a slice of RBD image snapshots that are part of a\ngroup snapshot.\n\nImplements:\n\n\tint rbd_group_snap_get_info(rados_ioctx_t group_p,\n\t                        const char *group_name,\n\t                        const char *snap_name,\n\t                        rbd_group_snap_info2_t *snaps);\n",
         "added_in_version": "v0.30.0",
         "expected_stable_version": "v0.32.0"
+      },
+      {
+        "name": "Image.EncryptionLoad2",
+        "comment": "EncryptionLoad2 enables IO on an open encrypted image and ancestor images.\nThe first EncryptionOptions in the slice is applied to the image, the second\nto the first ancestor, the third to the second ancestor and so on.\nIf the length of the slice is smaller than the number of ancestors the\nfinal item in the slice will be applied to all remaining ancestors.\n\nImplements:\n\n\tint rbd_encryption_load2(rbd_image_t image,\n\t                         const rbd_encryption_spec_t *specs,\n\t                         size_t spec_count);\n",
```

Comment:
> and ancestor images.

I/O on ancestor images is impossible.  The difference between `EncryptionLoad` and `EncryptionLoad2` lies only in how ancestor images are opened (which encryption spec is used when).

> the final item in the slice will be applied to all remaining ancestors.

... but if an ancestor for which the final item is being reused turns out to not match any encryption format known to librbd, the ancestor is interpreted as plaintext.